config/webpacker.yml

```diff
@@ -68,7 +68,7 @@ development:
     port: 3035
     # Hot Module Replacement updates modules while the application is running without a full reload
     # Used instead of the `hot` key in https://webpack.js.org/configuration/dev-server/#devserverhot
-    hmr: false
+    hmr: true
```

Comment:
**question:** Did this appear to actually result in hot module replacement? (I'd be pleasantly surprised if it did. Either way, no real harm in toggling this to `true`, I suppose.)

Reply:
I don't think it works properly, i'll revert to `false` to avoid confusion